subgraphs/badger-dao/config/matic.json

```diff
@@ -1,7 +1,29 @@
 {
   "chain": "matic",
   "registry": {
-    "address": "0x22765948A3d5048F3644b81792e4E1aA7ea3da4a",
-    "startBlock": 17191330
-  }
+    "address": "0xfda7eb6f8b7a9e9fcfd348042ae675d1d652454f",
```

Comment:
new registry, had to add vaults now since newer registry was deployed after vaults